linux_os/guide/system/network/network-iptables/iptables_ruleset_modifications/ip6tables_rules_for_open_ports/rule.yml

```diff
@@ -0,0 +1,34 @@
+documentation_complete: true
+
+prodtype: ubuntu2204
+
+title: 'Ensure ip6tables firewall rules exist for all open ports'
```

Comment:
In alignment to [Style Guide](https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#rule-sections) it must be in [Title case](https://en.wikipedia.org/wiki/Title_case)